app/controllers/concerns/idv/verify_info_concern.rb

```diff
@@ -336,6 +336,8 @@ def move_applicant_to_idv_session
     end
 
     def add_proofing_costs(results)
+      return if results[:context][:sp_costs_added]
```

Comment:
Can you say more about the session not being trustworthy? This is looking at the result from the resolution proofing job. That is what we use to determine if proofing was successful so we _should_ be able to trust it.

Reply:
Oh wait, it is sometimes nil

Reply:
old values before we started recording will be nil, but hopefully new ones are not nil?

Reply:
Actually it is nil quite a bit. 

I don't really want to do an `add_sp_cost unless sp_cost_record_exists_for(transaction_id)` kind of thing because we'll have SP cost calculation code running in two separate contexts (request processing and worker). Those two things _might_ trample on one another in weird ways if we don't have the DB enforcing uniqueness.

Having the working add a flag that says "it's ok, I already took care of this" does an end-run around those concerns.

Reply:
(The flag will go away once we remove this code from `VerifyInfoConcern`)